src/test/java/com/exactpro/epfast/ByteBuffUtils.java

```diff
@@ -0,0 +1,27 @@
+package com.exactpro.epfast;
+
+import com.exactpro.epfast.decoder.OverflowException;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+
+import java.util.ArrayList;
+
+public class ByteBuffUtils {
+
+    private static ByteBuf fromHex(String hex) {
+        ByteBuf buf = Unpooled.buffer();
```

Comment:
rename buf to buffer. Ther is no point in abbreviations when they aren't really needed